packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/LightCurveViewer/LightCurveViewer.js

```diff
@@ -266,6 +300,66 @@ class LightCurveViewer extends Component {
     this.d3svg.call(addInterfaceLayer)
     this.d3interfaceLayer = this.d3svg.select('.interface-layer')
     this.d3interfaceLayer.call(this.zoom)
+    this.updateInteractionMode(props.interactionMode)
+  }
+  
+  /*
+  Updates interaction logic, switching between navigation and annotation.
+   */
+  updateInteractionMode(interactionMode = '') {
+    if (!this.zoom || !this.d3interfaceLayer) return
+    
+    if (interactionMode === 'annotate') {
+      // HACK: Prevent zoom by "running in place"
+      // this.zoom.on('zoom', null) doesn't work, because transforms are just
+      // "deferred" until Move Mode is reinstated.
+      /*this.savedTransform = this.getCurrentTransform()
+      this.zoom.on('zoom', () => {
+        if (d3.event.transform.x !== this.savedTransform.x
+            || d3.event.transform.y !== this.savedTransform.y
+            || d3.event.transform.k !== this.savedTransform.k) {
+          this.zoom.transform(this.d3interfaceLayer, this.savedTransform)
+        }
+      })
+      this.d3interfaceLayer.style('cursor', 'crosshair')
+      this.d3interfaceLayer.on('click', this.doInsertAnnotation.bind(this))*/
+      
+      this.d3svg.on('click', this.doInsertAnnotation.bind(this))
+      this.d3interfaceLayer.style('display', 'none')
+      
+    } else if (interactionMode === 'move') {
+      //this.zoom.on('zoom', this.doZoom.bind(this))
+      
+      this.d3svg.on('click', null)
+      this.d3interfaceLayer.style('display', null)
```

Comment:
I don't think `display: null` is valid CSS... I'm a little confused about what's going on here.

Reply:
Calling `d3.selection.style('cssAttribute', null)` basically _removes_ the style property altogether. (See note [here](https://github.com/d3/d3-selection#selection_style))

In this case, I am removing the value of `interfaceLayer.style.display` (set in `if (interactionMode === 'annotate')` to be `display: none`) so it returns to its browser-default value ('inline', IIRC). Basically, it makes the Interface Layer "visible" again. 

(Additional note: it's `d3.selection.style('cssAttribute', null)` , not `d3.selection.style('cssAttribute', 'null')` - accidentally turning that `null` into a string _did_ screw me over once before.)

Reply:
Ok, I think it'd be more clear to set it to what you intend, i.e. `display: inline` rather than null here.

Reply:
Can do 👌